docs/src/api.md

```diff
@@ -1,7 +1,7 @@
 # [**12.** API Reference](@id API-Reference)
 
-This section describes the function interface, or "API mode",
-for interacting with Pkg.jl. The function API is recommended
+This section describes the functional interface,
+for interacting with Pkg.jl. The "functional API" is recommended
```

Comment:
```suggestion
This section describes the functional API interface for interacting with Pkg.jl.
It is recommended to use the functional API, rather than the Pkg REPL mode,
```

Reply:
Do you want to keep "API interface"? It's not uncommon to have those redundancies (e.g. LCD display), but I tried to avoid it.

Reply:
updated